### PR TITLE
Mark node offline if cannot delete workspace

### PR DESCRIPTION
**Pull request by llxia**
related: https://github.com/adoptium/aqa-tests/issues/5358

### CODE REVIEW

**Review by LongyuZhang — APPROVED** [review]
LGTM

File: buildenv/jenkins/openjdk_tests [hunk 1]
```diff
@@ -499,6 +499,7 @@ def runTest() {
                     errorList.add(".*java.io.IOException: Cannot run program \"nohup\".*");
                     errorList.add(".*unzip: command not found.*");
                     errorList.add(".*ant: command not found.*");
+                    errorList.add(".*ERROR: Cannot delete workspace :Unable to delete .*");
```
Comment (karianna) [inline buildenv/jenkins/openjdk_tests:502]: Is that 2nd colon in the right place?  Not sure if we're matching exact strings.

Comment (llxia) [reply buildenv/jenkins/openjdk_tests:502]: The message is correct. However, I will update it to accept space in case this gets fixed by Jenkins.

**Review by karianna — APPROVED** [review]

**Comment by pshipton** [comment]
Seems this has tanked all the OpenJ9 testing. Every test run is failing with the following.

Example https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_ppc64_aix_OMR_testList_0/559/
```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: 502: unexpected char: '\' @ line 502, column 70.
   ROR: Cannot delete workspace :\s?Unable 
```

**Comment by pshipton** [comment]
Created a revert PR https://github.com/adoptium/aqa-tests/pull/5386

**Pull request merged**